src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseMultiSignMessage.cs

```diff
@@ -598,16 +558,16 @@ public override int GetEncodedLength()
         /// Attempts to encode this message into the specified buffer.
         /// </summary>
         /// <param name="destination">The buffer in which to write the encoded value.</param>
-        /// <param name="bytesWritten">On success, receives the number of bytes written to <paramref name="destination" />.</param>
-        /// <returns><see langword="true" /> if <paramref name="destination" /> had sufficient length to receive the value; otherwise, <see langword="false" />.</returns>
+        /// <param name="bytesWritten">On success, receives the number of bytes written to <paramref name="destination"/>.</param>
+        /// <returns><see langword="true"/> if <paramref name="destination"/> had sufficient length to receive the value; otherwise, <see langword="false"/>.</returns>
         /// <remarks>Use <see cref="GetEncodedLength()"/> to determine how many bytes result in encoding this message.</remarks>
         /// <exception cref="InvalidOperationException">
         ///   <para>
-        ///     The <see cref="CoseMessage.ProtectedHeaders" /> and <see cref="CoseMessage.UnprotectedHeaders" /> collections have one or more labels in common.
+        ///     The <see cref="CoseMessage.ProtectedHeaders"/> and <see cref="CoseMessage.UnprotectedHeaders"/> collections have one or more labels in common.
         ///   </para>
         ///   <para>-or-</para>
         ///   <para>
-        ///     Message must carry at least one signature.
+        ///     Message does not contain at least one signature.
```

Comment:
"Message" should be "The message" so that it works with the prefix "[this method] Throws InvalidOperationException if"

```suggestion
        ///     The message does not contain at least one signature.
```